packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -231,11 +231,15 @@ hideModels.value = props.configuration.hideModels ?? false
 useDeprecationWarnings(props.configuration)
 </script>
 <template>
-  <Style>{{
-    getThemeStyles(configuration.theme, {
-      fonts: configuration.withDefaultFonts,
-    })
-  }}</Style>
+  <!-- Beware: This does NOT work in SSR -->
```

Comment:
cc @hwkr

Reply:
Tbh I'm not really sure how to fix this, I thought originally @hanspagel did it this way so it _would_ support SSR.